### PR TITLE
Typography: @next foundation typography

### PR DESCRIPTION
**Pull request by ninariccimarie**
[Jira](https://glints.atlassian.net/browse/SS-723)
[Figma](https://www.figma.com/file/87fhOdF8vk1Uiy20bCqS85/Sirius-Web---UI-Kit-3.0-%5BQ4-2022%5D?node-id=1%3A10&t=mVJNCW2A5yOdlBUF-0)
[Storybook](https://5fbcb813bd13c900214bb971-stcmtmbhsa.chromatic.com/?path=/docs/typography-next-typography--interactive)

## Storybook 

Screenshot 2022-12-20 at 11 40 06 AM

### CODE REVIEW

File: src/General/@next/Typography/TypographyStyles.ts [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  letter-spacing: 0;
+
+  @media (max-width: ${Breakpoints.large}) {
+    font-weight: 600;
+    font-size: 20px;
+  }
+`;
+
+const headline6 = css`
+  font-family: Noto Sans TC, Noto Sans SC, Noto Sans KR, Noto Sans JP, Noto Sans,
```
Comment (marthayaputra) [inline src/General/@next/Typography/TypographyStyles.ts]: can we put the font family in another var?

File: src/General/@next/Typography/TypographyStyles.ts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+  ['body1']: body1,
+  ['body2']: body2,
+  ['button']: button,
+  ['caption']: caption,
+  ['overline']: overline,
+};
+
+export const StyledTypography = styled.p<TypographyProps>`
+  color: ${props => props.color};
+  ${props => variantCssMapping[props.variant]}
```
Comment (marthayaputra) [inline src/General/@next/Typography/TypographyStyles.ts:198]: Is it possible to put a warning in console if the variant passed not found? That way it'll help devs to figure out faster

Comment (ninariccimarie) [reply src/General/@next/Typography/TypographyStyles.ts:198]: agree, we'll probably need it if it's used within non typescript

File: src/General/@next/Typography/TypographyStyles.ts (hunk 1 above)
Comment (ninariccimarie) [reply src/General/@next/Typography/TypographyStyles.ts]: @marthayaputra addressed

File: src/General/@next/Typography/TypographyStyles.ts (hunk 2 above)
Comment (ninariccimarie) [reply src/General/@next/Typography/TypographyStyles.ts:198]: @marthayaputra here's how the warning should look like if a variant passed is not found
Screenshot 2022-12-21 at 1 34 34 PM
Screenshot 2022-12-21 at 2 38 21 PM

**Review by marthayaputra — APPROVED** [review]

**Pull request merged**